src/Slider/slider.css

```diff
@@ -345,6 +345,11 @@
 
 .disabled
 {
+    cursor: not-allowed;
+    &  *
+    {
+      cursor: not-allowed;
```

Comment:
We are not currently using `not-allowed` anywhere. This should be `cursor: default` to be consistent with other components.

Reply:
Ah ok 👍

Reply:
Done 👍